examples/lock-app/efr32/include/DataModelHandler.h

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ * @file
+ *   This file defines the API for the handler for data model messages.
+ */
+
+#ifndef DATA_MODEL_HANDLER_H
+#define DATA_MODEL_HANDLER_H
+
+namespace chip {
```

Comment:
Would it be worth it to include the header rather than having a forward declaration? Or maybe drop this header alltogether since others can just include the packet buffer header instead (and cpp files probably need it anyway)